server/odc-service/src/main/java/com/oceanbase/odc/service/db/schema/syncer/object/DBExternalTableSyncer.java

```diff
@@ -41,13 +46,17 @@
  * @since: 4.3.3
  */
 @Component
+@Slf4j
 public class DBExternalTableSyncer extends AbstractDBObjectSyncer<TableExtensionPoint> {
 
     @Autowired
     private PermissionRepository permissionRepository;
 
     @Autowired
     private UserPermissionRepository userPermissionRepository;
+    @Autowired
+    @Lazy
+    private TableService tableService;
```

Comment:
why `Lazy`?

Reply:
loop dependency

Reply:
we may solve this problem as follow:
1. add a new method  in VersionDiffService, named `isExternalSupported`
2. ref this method in bot tableservice and here

circle ref loop is not allowed